### ISSUE TITLE
Some UI tweaks...

### ISSUE THREAD

**Issue by ChapterGrim** [fmasa/wfrp-master#86]
It'd be nice if the information in the "Edit Character" section and "tabs" conformed, that way information was visible and clearly arrayed in tidier ergonomic layout.

The facility to see the text of Conditions in the "tab", which would likely apply to Psychology options too; psychologies might benefit from something akin to the layout off "Skills" "tab" or similar...

The facility to see which skills are Basic and Advanced at a glance, and possibly a way to list native languages distinct from the other skills, there's some utility around this with how skills like Sail and the Linguistics Talent work too; the ability to clearly distinguish between in-Career, Non-Career, and Species Advances.

A similar issue exists with Spells, as it's not at all possible for a player to see which Lore a spell belongs to, and the Lore features aren't accessible which would be handy. Perhaps grouping them by Lore as with "Skills, Talents, and Traits" in that "tab"?

**Comment by fmasa** [comment]
> It'd be nice if the information in the "Edit Character" section and "tabs" conformed, that way information was visible and clearly arrayed in tidier ergonomic layout.

Yes, this would certainly be one way to do this. The current state results from various redesigns and I agree that we can do better. I'll think about this more.

> The facility to see the text of Conditions in the "tab", which would likely apply to Psychology options too; psychologies might benefit from something akin to the layout off "Skills" "tab" or similar...

This would require import from Core Rulebook. Certainly possible, but quite a lot of work. The upside would be support for additional Conditions (*Rattled*)

> The facility to see which skills are Basic and Advanced at a glance, and possibly a way to list native languages distinct from the other skills, there's some utility around this with how skills like Sail and the Linguistics Talent work too;

I'm totally for some visual distinction for Basic and Advanced Skills, this is really useful for Assistance. Do you have something in mind? Maybe a bold text or icon?

> the ability to clearly distinguish between in-Career, Non-Career, and Species Advances.

For Species advances there is currently no way to know as app does not have a concept of Species Skills/Talents. For Career advances I currently plan separate screen intended for Career all advances (advancing Skills, Talents and Characteristics).
Is there some mechanic that would benefit from tracking Species Advances separately?

> A similar issue exists with Spells, as it's not at all possible for a player to see which Lore a spell belongs to, and the Lore features aren't accessible which would be handy. Perhaps grouping them by Lore as with "Skills, Talents, and Traits" in that "tab"?

Great idea, I can see how current state can be confusing from spellcasters with multiple lores + petty spells + arcane spells.

**Comment by fmasa** [comment]
Anyway, moving this to discussions. I will extract the actionable parts to Issues to make clear what has already been improved :slightly_smiling_face:
